barretenberg/cpp/src/barretenberg/vm/tests/avm_arithmetic.test.cpp

```diff
@@ -720,6 +808,23 @@ TEST_F(AvmArithmeticTestsU8, nonEquality)
     validate_trace(std::move(trace));
 }
 
+// Test correct division of U8 elements using faster method
```

Comment:
There is no concept of fast division anymore. We have standard or a < b. Here, this is the standard one.